microk8s-resources/wrappers/run-kubelite-with-args

```diff
@@ -125,6 +125,20 @@ then
   fi
 fi
 
+if ! is_strict &&
+   (systemctl show snap.microk8s.daemon-kubelite.service -p NRestarts | grep -qv "NRestarts=0") &&
+   (grep -qv cpuset /sys/fs/cgroup/cgroup.subtree_control) &&
+   [ ! -e /etc/systemd/system/snap.microk8s.daemon-kubelite.service.d/delegate.conf ]
+then
+  mkdir -p /etc/systemd/system/snap.microk8s.daemon-kubelite.service.d
+  tee /etc/systemd/system/snap.microk8s.daemon-kubelite.service.d/delegate.conf > /dev/null <<EOF
+[Service]
+Delegate=yes
+EOF
+  systemctl daemon-reload
+  snap restart microk8s
```

Comment:
is there any situation where these might fail and cause the kubelite process to start crash-looping?

Reply:
Good point, not sure so let me add ` || true` to make sure we just skip on failures here.